conf/script/build-cortx-ha

```diff
@@ -77,6 +90,9 @@ init() {
 }
 
 cleanup() {
+    echo "Delete alert"
+    pcs alert show | grep ha_iem_event && pcs alert remove ha_iem_event
```

Comment:
Please add a comment that it will remove ha_iem_recipient too.

Reply:
Done